### PR TITLE
Release v11.5.0

### PR DESCRIPTION
**Pull request by danj-stripe**
- [x] Update README (if necessary)
- [x] Update CHANGELOG with any new features or breaking changes
- [x] ~~Update MIGRATING (if necessary)~~
- [x] Update STPSDKVersion in STPAPIClient.h
- [x] Update version in Info.plist (make sure to not change the "InfoDictionary" field)
- [x] Update VERSION
- [x] Update `Stripe.podspec` (make sure there aren't any new directories added, and if so, add them to the "files" section
- [x] Update documentation: `./ci_scripts/build_documentation.sh`
- [x] Make sure `pod lib lint` succeeds without warnings or errors
  - There is a single, expected, warning, due to [#823](https://github.com/stripe/stripe-ios/pull/823)
- [x] Make sure `./ci_scripts/export_builds.sh` succeeds without warnings or errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +50,7 @@ To try out any of these examples, clone this repository, open `Stripe.xcworkspac
 1. Execute `./setup.sh` from the root of the repository to build the necessary dependencies.
 2. If you haven't already, sign up for a [Stripe account](https://dashboard.stripe.com/register) (it takes seconds). Then go to https://dashboard.stripe.com/account/apikeys.
 3. Replace the `stripePublishableKey` constant in CheckoutViewController.swift (for the Standard Integration app) or Constants.m (for the Custom Integration app) with your Test Publishable Key.
-4. Head to https://github.com/stripe/example-ios-backend/tree/v11.0.0 and click "Deploy to Heroku" (you may have to sign up for a Heroku account as part of this process). Provide your Stripe test secret key for the STRIPE_TEST_SECRET_KEY field under 'Env'. Click "Deploy for Free".
+4. Head to https://github.com/stripe/example-ios-backend and click "Deploy to Heroku" (you may have to sign up for a Heroku account as part of this process). Provide your Stripe test secret key for the STRIPE_TEST_SECRET_KEY field under 'Env'. Click "Deploy for Free".
```
Comment (bdorfman-stripe) [inline README.md]: Why did you remove the versioned link from the backend url?

Comment (danj-stripe) [reply README.md]: @ksun-stripe made updates to the example-ios-backend, and that tag isn't current any more. We could also re-tag it and update the link to the new tag, but I wasn't sure why we wouldn't just want to link people to `master`

Comment (bdorfman-stripe) [reply README.md]: Ah we specifically wanted to start making tagged releases for the backend and having the tagged releases here match up to the version of the backend that supports this version of the sdk. Otherwise you can get confusing version mismatches (eg latest master on backend might not match the latest tagged release of the sdk. Or if you are on an old version you can't easily figure out what version of the backend you need). 

If Kathy has made new changes to the backend that we want people to be using with 11.5.0 we should create a new tagged version for the backend and link that here instead.

**Comment by bdorfman-stripe** [comment]
LGTM
